mono/metadata/object-offsets.h

```diff
@@ -88,10 +88,17 @@ DECL_OFFSET(MonoDelegate, method_code)
 DECL_OFFSET(MonoDelegate, method_is_virtual)
 DECL_OFFSET(MonoDelegate, extra_arg)
 
+#ifdef ENABLE_NETCORE
```

Comment:
nit: A `typedef` could clean this and the other up, albeit maybe less clear due to abstraction.
````
#ifdef ENABLE_NETCORE
typedef MonoThread MonoInternalThread;
#endif 
```